### ISSUE TITLE
libraw (needed by freeimage) failed to compile (BUILD_FAILED)

### ISSUE THREAD

**Issue by kwach** [Microsoft/vcpkg#1665]
* I'm using Windows 10 Home 
* Repro steps:
`vcpkg install freeimage`
* Expected: package and deps installs just fine
* Actual outcome: `libraw` failed to install

Error details:
* Package: libraw:x86-windows
* Vcpkg version: 0.0.83-45d9d68a5825c78d71d8508ff9554d2552049138

```
-- Build x86-windows-rel
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:43 (message):
    Command failed: C:/Users/x/Documents/workspace/vcpkg/downloads/cmake-3.9.1-win32-x86/bin/cmake.exe;--build;.;--config;Release;--;/p:VCPkgLocalAppDataDisabled=true;/p:UseIntelMKL=No;/m
    Working Directory: C:/Users/x/Documents/workspace/vcpkg/buildtrees/libraw/x86-windows-rel
    See logs for more information:
      C:\Users\x\Documents\workspace\vcpkg\buildtrees\libraw\build-x86-windows-rel-out.log
      C:\Users\x\Documents\workspace\vcpkg\buildtrees\libraw\build-x86-windows-rel-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:27 (vcpkg_execute_required_process)
  ports/libraw/portfile.cmake:30 (vcpkg_build_cmake)
  scripts/ports.cmake:72 (include)
```

Logfile say:
```
           Creating library C:/Users/x/Documents/workspace/vcpkg/buildtrees/libraw/x86-windows-rel/Release/raw.lib and object C:/Users/x/Documents/workspace/vcpkg/buildtrees/libraw/x86-win
dows-rel/Release/raw.exp
     7>dcraw_fileio.obj : error LNK2019: unresolved external symbol _cmsOpenProfileFromFile@8 referenced in function "protected: void __thiscall LibRaw::apply_profile(char const *,char const *)" (?apply_
profile@LibRaw@@IAEXPBD0@Z) [C:\Users\x\Documents\workspace\vcpkg\buildtrees\libraw\x86-windows-rel\raw.vcxproj]
     7>dcraw_fileio.obj : error LNK2019: unresolved external symbol _cmsOpenProfileFromMem@8 referenced in function "protected: void __thiscall LibRaw::apply_profile(char const *,char const *)" (?apply_p
rofile@LibRaw@@IAEXPBD0@Z) [C:\Users\x\Documents\workspace\vcpkg\buildtrees\libraw\x86-windows-rel\raw.vcxproj]
     7>dcraw_fileio.obj : error LNK2019: unresolved external symbol _cmsCloseProfile@4 referenced in function "protected: void __thiscall LibRaw::apply_profile(char const *,char const *)" (?apply_profile
@LibRaw@@IAEXPBD0@Z) [C:\Users\x\Documents\workspace\vcpkg\buildtrees\libraw\x86-windows-rel\raw.vcxproj]
     7>dcraw_fileio.obj : error LNK2019: unresolved external symbol _cmsCreate_sRGBProfile@0 referenced in function "protected: void __thiscall LibRaw::apply_profile(char const *,char const *)" (?apply_p
rofile@LibRaw@@IAEXPBD0@Z) [C:\Users\x\Documents\workspace\vcpkg\buildtrees\libraw\x86-windows-rel\raw.vcxproj]
     7>dcraw_fileio.obj : error LNK2019: unresolved external symbol _cmsCreateTransform@24 referenced in function "protected: void __thiscall LibRaw::apply_profile(char const *,char const *)" (?apply_pro
file@LibRaw@@IAEXPBD0@Z) [C:\Users\x\Documents\workspace\vcpkg\buildtrees\libraw\x86-windows-rel\raw.vcxproj]
     7>dcraw_fileio.obj : error LNK2019: unresolved external symbol _cmsDeleteTransform@4 referenced in function "protected: void __thiscall LibRaw::apply_profile(char const *,char const *)" (?apply_prof
ile@LibRaw@@IAEXPBD0@Z) [C:\Users\x\Documents\workspace\vcpkg\buildtrees\libraw\x86-windows-rel\raw.vcxproj]
     7>dcraw_fileio.obj : error LNK2019: unresolved external symbol _cmsDoTransform@16 referenced in function "protected: void __thiscall LibRaw::apply_profile(char const *,char const *)" (?apply_profile
@LibRaw@@IAEXPBD0@Z) [C:\Users\x\Documents\workspace\vcpkg\buildtrees\libraw\x86-windows-rel\raw.vcxproj]
     7>C:\Users\x\Documents\workspace\vcpkg\buildtrees\libraw\x86-windows-rel\Release\raw.dll : fatal error LNK1120: 7 unresolved externals [C:\Users\x\Documents\workspace\vcpkg\buildtrees\
libraw\x86-windows-rel\raw.vcxproj]
     7>Done Building Project "C:\Users\x\Documents\workspace\vcpkg\buildtrees\libraw\x86-windows-rel\raw.vcxproj" (default targets) -- FAILED.
     5>Done Building Project "C:\Users\x\Documents\workspace\vcpkg\buildtrees\libraw\x86-windows-rel\4channels.vcxproj" (default targets) -- FAILED.
```

Note: dbg built just fine.

I'm using:
Microsoft Visual Studio Community 2017 
Version 15.3.1
VisualStudio.15.Release/15.3.1+26730.8
Microsoft .NET Framework
Version 4.7.02046

**Comment by Mixaill** [comment]
@kwach fixed in #1666

**Comment by ras0219-msft** [comment]
Fixed in #1666.
